developer/src/kmc-kmn/src/compiler/kmn-compiler-messages.ts

```diff
@@ -40,71 +40,165 @@ export const enum KmnCompilerMessageRanges {
   RANGE_CompilerMessage_Max = 0xFFF, // Highest available error code for kmc-kmn
 }
 
-/*
-  The messages in this class share the namespace with messages from
-  kmn_compiler_errors.h, which are generated by kmcmplib, so values below 0x900
-  are reserved for kmcmplib messages.
-*/
+/**
+ * @public
+ * Error messages reported by the .kmn compiler. The messages in this class
+ * share the namespace with messages from kmn_compiler_errors.h, which are
+ * generated by kmcmplib, so values below 0x900 are reserved for kmcmplib
+ * messages.
```

Comment:
```suggestion
 * @public
 * Error messages reported by the .kmn compiler. The error codes in this class
 * share an address space with messages from kmn_compiler_errors.h, which are
 * generated by kmcmplib, so codes below 0x900 are reserved for kmcmplib
 * messages.
```

(yes I'm in spec pedantic mode!) It's more of a code space than an address space, but most important is consistent wording. So ignore if needed.